ods_ci/tests/Resources/Files/triton/triton_keras_rest_servingruntime.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
We are adding this file because to test it on local without those files we cannot test them , once the all PR's are approved we remove the all the duplicate files

Reply:
then you can just merge one PR with the file you need and then rebase the dependent one